assets/blueprints/blueprint.json

```
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"landingPage": "/wp-admin/site-editor.php",
+	"plugins": [ "gutenberg" ],
+	"login": true,
+	"features": {
+		"networking": true
+	},
+	"preferredVersions": {
+		"php": "8.0",
```

Is there any downside to using a PHP version of "latest"? If we can do that, maintainers won't have to worry about updating this Blueprint as PHP versions are retired.

```suggestion
		"php": "latest",
```

---

Good point. I will change it.